images/microOS/config.xml

```diff
@@ -1,551 +1,102 @@
-            editbootinstall="editbootinstall_rpi.sh"
-            btrfs_root_is_readonly_snapshot="true"
-            btrfs_quota_groups="false"
-        >
-            <systemdisk>
-                <volume name="home"/>
-                <volume name="root"/>
-                <volume name="opt"/>
-                <volume name="srv"/>
-                <volume name="boot/grub2/arm64-efi" mountpoint="boot/grub2/arm64-efi"/>
-                <volume name="boot/writable"/>
-                <volume name="usr/local"/>
-                <volume name="var" copy_on_write="false"/>
-            </systemdisk>
-        </type>
     </preferences>
-    <preferences profiles="RaspberryPi2">
-        <version>16.0.0</version>
-        <packagemanager>zypper</packagemanager>
-        <bootloader-theme>openSUSE</bootloader-theme>
-        <rpm-excludedocs>true</rpm-excludedocs>
-        <locale>en_US</locale>
+    <preferences profiles="Vagrant-x86_64">
         <type
             image="oem"
-            initrd_system="dracut"
-            filesystem="btrfs"
-            fsmountoptions="noatime,compress=lzo"
-            bootloader="grub2"
-            bootloader_console="gfxterm"
-            firmware="efi"
-            kernelcmdline="quiet systemd.show_status=yes console=ttyAMA0,115200 console=tty0 net.ifnames=0 \$ignition_firstboot ignition.platform.id=metal"
-            bootpartition="false"
-            devicepersistency="by-uuid"
-            btrfs_root_is_snapshot="true"
-            efipartsize="16"
-            editbootinstall="editbootinstall_rpi.sh"
-            btrfs_root_is_readonly_snapshot="true"
-            btrfs_quota_groups="false"
-        >
-            <systemdisk>
-                <volume name="home"/>
-                <volume name="root"/>
-                <volume name="opt"/>
-                <volume name="srv"/>
-                <volume name="boot/grub2/arm-efi" mountpoint="boot/grub2/arm-efi"/>
-                <volume name="boot/writable"/>
-                <volume name="usr/local"/>
-                <volume name="var" copy_on_write="false"/>
-            </systemdisk>
-        </type>
-    </preferences>
-    <preferences profiles="Vagrant_x86_64">
-        <version>16.0.0</version>
-        <packagemanager>zypper</packagemanager>
-        <bootloader-theme>openSUSE</bootloader-theme>
-        <rpm-excludedocs>true</rpm-excludedocs>
-        <locale>en_US</locale>
-        <type
-            image="vmx"
-            filesystem="btrfs"
+            filesystem="xfs"
             format="vagrant"
             bootloader="grub2"
             bootloader_console="gfxterm"
             firmware="uefi"
-            kernelcmdline="quiet systemd.show_status=yes console=ttyS0,115200 console=tty0 net.ifnames=0 \$ignition_firstboot ignition.platform.id=metal"
+            kernelcmdline="quiet systemd.show_status=yes console=ttyS0,115200 console=tty0 net.ifnames=0 \$ignition_firstboot ignition.platform.id=metal root=/dev/vda1 disk=/dev/vda"
```

Comment:
this line is the culprit, and why it doesn't work with vagrant-libvirt. Please drop the ` root=/dev/vda1 disk=/dev/vda` at the end.

Reply:
this is also interesting .. because I don't think I touch this line and not sure why it is failing now. It should be the same the image before. But I drop them and see it helps.